src/less/components/topbar.less

```diff
@@ -328,7 +328,7 @@
             background-color: transparent;
 
             /* padding-top-bottom, padding-left-right, font-size, line-height */
-            .button-size(0.5rem; 0.5rem; 1rem; 1rem; @btn-border-radius-base;);
+            //.button-size(0.5rem; 0.5rem; 1rem; 1rem; @btn-border-radius-base;); IS THIS NEEDED?
```

Comment:
if not needed then let's remove it
(tested & compared visually `.topbar-close` and it seems fine, identical to main branch)